src/components/manage/Blocks/Block/BlocksForm.jsx

```diff
@@ -32,6 +32,7 @@ const BlocksForm = (props) => {
     manage,
     children,
     blocksConfig = config.blocks.blocksConfig,
+    editable,
```

Comment:
can we do `editable=true` here as default? Otherwise we'll have a ton of code to migrate, we use BlocksForm in other places as well.

Reply:
yes sure